src/components/Home/ClusterDashboardItem.js

```diff
@@ -251,6 +254,22 @@ function ClusterDashboardItem({
               </span>
             </RefreshableLabel>
             <Dot style={{ paddingLeft: 0 }} />
+            <RefreshableLabel value={cluster.kubernetes_version}>
+              <span>
+                <i className='fa fa-kubernetes' title='Kubernetes version' />{' '}
+                <span>
+                  {(() => {
+                    const kubernetes = release.components.find(
+                      (component) => component.name === 'kubernetes'
+                    );
+                    if (kubernetes) return kubernetes.version;
+
+                    return null;
+                  })()}
```

Comment:
Can you please extract this into a separate function, this is a pattern that we're trying to get rid of?

Reply:
Gerald already mentioned that it can be replaced simply with `release.kubernetesVersion`.